contracts/gas-snapshots/keystone.gas-snapshot

```diff
@@ -78,22 +78,22 @@ CapabilityRegistry_UpdateNodesTest:test_RevertWhen_P2PIDEmpty() (gas: 27313)
 CapabilityRegistry_UpdateNodesTest:test_RevertWhen_SignerAddressEmpty() (gas: 26996)
 CapabilityRegistry_UpdateNodesTest:test_RevertWhen_UpdatingNodeWithoutCapabilities() (gas: 25560)
 CapabilityRegistry_UpdateNodesTest:test_UpdatesNodeInfo() (gas: 158847)
-KeystoneForwarder_ReportTest:test_Report_ConfigVersion() (gas: 2676465)
-KeystoneForwarder_ReportTest:test_Report_SuccessfulDelivery() (gas: 150738)
-KeystoneForwarder_ReportTest:test_RevertWhen_AlreadyProcessed() (gas: 158024)
-KeystoneForwarder_ReportTest:test_RevertWhen_AnySignatureIsInvalid() (gas: 111634)
-KeystoneForwarder_ReportTest:test_RevertWhen_AnySignerIsInvalid() (gas: 143377)
-KeystoneForwarder_ReportTest:test_RevertWhen_ReportHasDuplicateSignatures() (gas: 119553)
-KeystoneForwarder_ReportTest:test_RevertWhen_ReportHasIncorrectDON() (gas: 98500)
-KeystoneForwarder_ReportTest:test_RevertWhen_ReportHasInexistentConfigVersion() (gas: 98757)
-KeystoneForwarder_ReportTest:test_RevertWhen_ReportIsMalformed() (gas: 67777)
-KeystoneForwarder_ReportTest:test_RevertWhen_TooFewSignatures() (gas: 77819)
-KeystoneForwarder_ReportTest:test_RevertWhen_TooManySignatures() (gas: 78543)
+KeystoneForwarder_ReportTest:test_Report_ConfigVersion() (gas: 1874910)
+KeystoneForwarder_ReportTest:test_Report_SuccessfulDelivery() (gas: 148306)
+KeystoneForwarder_ReportTest:test_RevertWhen_AlreadyProcessed() (gas: 135334)
+KeystoneForwarder_ReportTest:test_RevertWhen_AnySignatureIsInvalid() (gas: 109559)
+KeystoneForwarder_ReportTest:test_RevertWhen_AnySignerIsInvalid() (gas: 141302)
+KeystoneForwarder_ReportTest:test_RevertWhen_ReportHasDuplicateSignatures() (gas: 117478)
+KeystoneForwarder_ReportTest:test_RevertWhen_ReportHasIncorrectDON() (gas: 76242)
+KeystoneForwarder_ReportTest:test_RevertWhen_ReportHasInexistentConfigVersion() (gas: 76499)
+KeystoneForwarder_ReportTest:test_RevertWhen_ReportIsMalformed() (gas: 45519)
+KeystoneForwarder_ReportTest:test_RevertWhen_TooFewSignatures() (gas: 55561)
+KeystoneForwarder_ReportTest:test_RevertWhen_TooManySignatures() (gas: 56285)
 KeystoneForwarder_SetConfigTest:test_RevertWhen_ExcessSigners() (gas: 20162)
 KeystoneForwarder_SetConfigTest:test_RevertWhen_FaultToleranceIsZero() (gas: 90143)
 KeystoneForwarder_SetConfigTest:test_RevertWhen_InsufficientSigners() (gas: 14511)
 KeystoneForwarder_SetConfigTest:test_RevertWhen_NotOwner() (gas: 90874)
-KeystoneForwarder_SetConfigTest:test_RevertWhen_ProvidingDuplicateSigners() (gas: 847644)
-KeystoneForwarder_SetConfigTest:test_SetConfig_FirstTime() (gas: 2224138)
-KeystoneForwarder_SetConfigTest:test_SetConfig_WhenSignersAreRemoved() (gas: 2365984)
+KeystoneForwarder_SetConfigTest:test_RevertWhen_ProvidingDuplicateSigners() (gas: 159111)
+KeystoneForwarder_SetConfigTest:test_SetConfig_FirstTime() (gas: 1535605)
+KeystoneForwarder_SetConfigTest:test_SetConfig_WhenSignersAreRemoved() (gas: 1808776)
```

Comment:
I wonder why you're getting different numbers for `setConfig` since there isn't any code changes there?

Reply:
https://github.com/smartcontractkit/chainlink/pull/13450/files#diff-c3709cf96d27e7fbf521a4b024277553a1346a48fdfa85a5567f86aecb629494L116

signers were set twice